packages/grid/_modules_/grid/hooks/features/keyboard/useGridKeyboardNavigation.ts

```diff
@@ -231,6 +237,14 @@ export const useGridKeyboardNavigation = (
           event.preventDefault(); // prevent Space event from scrolling
```

Comment:
```suggestion
          // prevent Space event from scrolling
```

Since `event.preventDefault()` is called below